TTIEX355P/TTIEX355P-IOC-01App/Db/ttiEX355P.db

```diff
@@ -214,6 +217,147 @@ record(bo, "$(P)RESET:SP")
 
 alias("$(P)RESET:SP", "$(P)RESET")
 
+
+# Current <-> Field conversion
+
+record(bo, "$(P)FIELD_CONV_DEFINED") {
+    field(DESC, "Whether field conversion is defined")
+    field(VAL, "$(FIELD_CONV_DEFINED)")
+    field(PINI, "YES")
+    field(ZNAM, "No")
+    field(ONAM, "Yes")
+}
+
+record(ao, "$(P)AMPSTOGAUSS"){
+    field(SCAN, "Passive")
+    field(DTYP, "Soft Channel")
+    field(VAL, "$(AMPSTOGAUSS)")
+    field(PINI, "YES")
+    field(EGU, "G/A")
+    field(PREC, "3")
+}
+
+record(calcout, "$(P)FIELD:SP:_CALC") {
+    field(INPA, "$(P)FIELD:SP")
+    field(INPB, "$(P)AMPSTOGAUSS")
+    field(CALC, "A/B")
+    field(OUT, "$(P)CURRENT:SP PP")
+}
+
+record(ao, "$(P)FIELD:SP") {
+    field(FLNK, "$(P)FIELD:SP:_CALC")
+    field(EGU, "G")
+    field(PREC, "3")
+    field(DESC, "field setpoint")
+    
```

Comment:
should we set the interest for this to high? or medium at least? This is settable through the OPI so might be nice for quick access